boards/arm/lpcxpresso55S69/doc/index.rst

```diff
@@ -0,0 +1,210 @@
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration only enables the first core for now and can
+be found in the following defconfig file:
+
+	``boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC55S69 SoC has IOCON registers, which can be used to configure the
+functionality of a pin.
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_29 | USART           | USART RX                   |
++---------+-----------------+----------------------------+
+| PIO0_30 | USART           | USART TX                   |
++---------+-----------------+----------------------------+
+| PIO1_4 _| GPIO            | RED LED                    |
++---------+-----------------+----------------------------+
+| PIO1_6  | GPIO            | BLUE_LED                   |
++---------+-----------------+----------------------------+
+| PIO1_7_ | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The LPC55S69 SoC is configured to use the internal FRO at 96MHz as a source for
+the system clock. Other sources for the system clock are provided in the SOC,
+depending on your system requirements.
+
+Serial Port
+===========
+
+The LPC55S69 SoC has 8 FLEXCOMM interfaces for serial communication.  One is
+configured as USART for the console and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the LPC-Link2 CMSIS-DAP Onboard Debug Probe,
+however the :ref:`pyocd-debug-host-tools` does not yet support this probe so you
+must reconfigure the board for one of the following debug probes instead.
+
+:ref:`lpclink2-jlink-onboard-debug-probe`
+-----------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`lpclink2-jlink-onboard-debug-probe` to program
+the J-Link firmware. Please make sure you have the latest firmware for this
+board.
+
+:ref: `opensda-daplink-onboard-debug-probe`
+-----------------------------------------
```

Comment:
Section heading underline needs to be at least as long as the title (add some more ``---``)